### ISSUE TITLE
MenuItem doesn't respond to first click (GTK)

### ISSUE THREAD

**Issue by piedar** [picoe/Eto#208]
I have a `MenuItem` built from a [custom command](https://github.com/piedar/MachoMap/blob/master/MachoMap/Commands/OpenFileCommand.cs).  I run my [program](https://github.com/piedar/MachoMap) on Linux with mono v3.2.8 and gtk-sharp v2.12.0.
- Expected behavior: Clicking Open brings up an `OpenFileDialog`.
- Actual behavior: Clicking Open alternates between doing nothing and bringing up an `OpenFileDialog` .

In other words, the `Command.OnExecuted` method does not fire on a single-click.
